lochness/mindlamp/__init__.py

```diff
@@ -183,26 +194,28 @@ def sync(Lochness: 'lochness.config',
                         dst_folder,
                         f'{subject_id}_{subject.study}_{data_name}_'
                         f'{date_str}_sound.mp3')
-                if is_transferred_and_removed(Lochness, sound_dst):
-                    continue
+                # if is_transferred_and_removed(Lochness, sound_dst):
+                    # continue
```

Comment:
There is a problem with Kevin's block commenting. Block comments should start with all `#` in the same column.

Reply:
4c4d7d0d39ee409deaae44492298781a8c728c7c